populator-machinery/controller.go

```diff
@@ -771,11 +771,6 @@ func (c *controller) syncPvc(ctx context.Context, key, pvcNamespace, pvcName str
 					// We'll get called again later when the pvcPrime gets bounded
 					return nil
 				}
-				pv, err := params.KubeClient.CoreV1().PersistentVolumes().Get(ctx, pvcPrime.Spec.VolumeName, metav1.GetOptions{})
-				if err != nil {
-					return err
-				}
-				params.PV = pv
```

Comment:
In what scenarios is params.PV non nil?

Reply:
This is the only place that we assign PV, remove PV from the `PopulatorParams` API